pywr/recorders/_recorders.pyx

```diff
@@ -1660,16 +1660,21 @@ cdef class AnnualTotalFlowRecorder(Recorder):
     For each scenario, record the total flow in each year across a list of nodes.
     Output from data property has shape: (years, scenario combinations)
 
+    A list of factors can be provided to scale the total flow (e.g. for calculating operational costs).
+
     Parameters
     ----------
     model : `pywr.core.Model`
     name : str
         The name of the recorder
     nodes : list
         List of `pywr.core.Node` instances to record
+    factors (OPTIONAL) : dictionary
```

Comment:
This is good. However, I would make it a list where each entry corresponds to a node in the `nodes` list. Accessing this will be much faster than the dictionary.

Reply:
This should also  be `factors : list, optional` as per numpy style guide : https://numpydoc.readthedocs.io/en/latest/format.html